### ISSUE TITLE
[Sourcing-2][Accessibility: ACC-264.1] List Item: Value count not read.

### ISSUE THREAD

**Issue by JyotiKumari1507** [SAP/fundamental-ngx#11285]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
Component having issue: https://sap.github.io/fundamental-ngx/#/core/select
Item/Value count i.e. ‘1 of 15 etc’ is not being read out, on navigating between the  drop-down using arrow keys.
Item count for  'Number' drop-down should be read to screen reader users

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
All latest version

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
Using Screen reader can check by implementing similar case. Just use any iSelect Dropdown. Item/Value count i.e. ‘1 of 15 etc’ will not being read out, on navigating between the  drop-down using arrow keys.

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations. 
Yes we have got this bug from Accessibility expert only.

#### Did you check the documentation and the API?
Yes

#### Is there anything else we should know?

**Comment by JyotiKumari1507** [comment]
@droshev Its defect as per me. Issue is when user slide down in list and can see completed list but not read out by screen reader after sliding down also. Please re-open this defect

**Comment by JyotiKumari1507** [comment]
@droshev use any dropdown using keyboard, sroll down it using keyboard down arrow. Screen reader is able to read only start few records which was visible in starting. Its not able to read which comes after scrolling down by keyboard. Attaching screenshot for same. We had syncup with Accessibility team today. As per them its defect, Please re-open this defect. Still any query please let me know we can have call
<img width="258" alt="image" src="https://github.com/SAP/fundamental-ngx/assets/157268666/2d2d0a98-c128-428a-aa90-dbcf4dcbd680">
<img width="296" alt="image" src="https://github.com/SAP/fundamental-ngx/assets/157268666/2d5be344-affc-4c35-9b74-dc69266ca0c4">

**Comment by JyotiKumari1507** [comment]
@nikolay-kolarov lets wait for recording by Accessibility team

**Comment by MeghanaVernekar** [comment]
Hello All,
Retested the issue and issue is resolved. Value count conveyed for list items. Evidence attached.
<img width="960" alt="EMC_Value count conveyed" src="https://github.com/SAP/fundamental-ngx/assets/172006721/2fbfeb3b-b56e-4c15-a78b-dee813b5bcc0">

Regards,
Meghana

**Comment by droshev** [comment]
Can you open a new ticket and creating a stackblitz with the issue?

**Comment by divyamalhotra488** [comment]
Hi @droshev, issue is fixed as mentioned by @MeghanaVernekar.
